java/core/src/test/org/apache/orc/TestVectorOrcFile.java

```diff
@@ -4133,7 +4133,7 @@ public void testZeroByteOrcFile(Version fileFormat) throws Exception {
     assertEquals(0, reader.getStripes().size());
     assertEquals(0, reader.getStatistics().length);
     assertEquals(0, reader.getMetadataKeys().size());
-    assertEquals(OrcFile.CURRENT_WRITER, reader.getWriterVersion());
+    assertEquals(OrcFile.WriterVersion.FUTURE, reader.getWriterVersion());
```

Comment:
Also, this looks like a wrong behavior change to me.

Reply:
Shouldn't the writer version of a zero byte orc file be unknown?

Reply:
No, it's defined as a valid file always. So, it should be the up-to-date version, which is `CURRENT_WRITER`.